### PR TITLE
187579400 update flash

### PR DESCRIPTION
**Pull request by charlienparker**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [x] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [x] Any endpoint modified in the PR only responds to the expected MIME types.
- [x] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [x] There are no inline styles added
- [x] There are no inline javascript added
- [x] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [x] Code does not use .html_safe
- [x] All images added/updated have alt text
- [x] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/story/show/187579400

# A brief description of the changes

Current behavior: The flash message as described in the ticket was using the old flash banner UI.

New behavior: The flash across all BS4 experiences now uses the new flash banner UI, and a new Severe banner was added.

Updated info banner for ticket:
Screenshot 2024-05-17 at 10 32 15 AM

Longer text:
Screenshot 2024-05-17 at 10 32 42 AM

New severe banner:
Screenshot 2024-05-17 at 12 04 38 PM

Flash in progress layout:
Screenshot 2024-05-21 at 1 55 09 PM

### CODE REVIEW

File: app/views/insured/interactive_identity_verifications/failed_validation.html.erb
Comment (charlienparker) [inline app/views/insured/interactive_identity_verifications/failed_validation.html.erb:1]: unrelated fix Diana and I found

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -348,30 +348,44 @@ def link_to_add_fields(name, f, association, classes = '')
...
+  def get_flash(use_bs4, type, msg)
+    type = use_bs4 ? get_bs4_flash_type(type) : type
     if is_announcement?(msg)
       render(:partial => 'layouts/announcement_flash', :locals => {:type => type, :message => msg[:announcement]})
     else
       render(:partial => 'layouts/flash', :locals => {:type => type, :message => msg})
     end
   end
 
+  def get_bs4_flash_type(type)
```
Comment (kristinmerbach) [inline app/helpers/application_helper.rb:376]: I wouldn't reference bs4 in the method name, this can just be `get_flash_type`

Comment (charlienparker) [reply app/helpers/application_helper.rb]: Fixed.

File: app/views/ui-components/bs4/v1/navs/_header.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <hidden id='dchbx_enroll_date_of_record' class='d-none'><%= TimeKeeper.date_of_record.iso8601 %></hidden>
 <header class="bg-white navbar-primary py-2 <%= 'pre_prod_nav_color' if ENV['ENROLL_REVIEW_ENVIRONMENT'] == true %>">
 
-  <nav class="navbar navbar-default navbar-expand-lg container py-1">
+  <nav class="navbar navbar-default navbar-expand-lg container py-1 px-0">
```
Comment (kristinmerbach) [inline app/views/ui-components/bs4/v1/navs/_header.html.erb:4]: rather than `py-1 px-0` you can do `p-0` which will take care of both vertical and horizontal padding

**Review by kristinmerbach — APPROVED** [review]

File: app/views/ui-components/bs4/v1/navs/_header.html.erb (hunk 2 above)
Comment (charlienparker) [reply app/views/ui-components/bs4/v1/navs/_header.html.erb:4]: so we're fine with removing the vertical padding then entirely? that would make the y padding 0 instead of 1

Comment (kristinmerbach) [reply app/views/ui-components/bs4/v1/navs/_header.html.erb:4]: I misread sorry! fine as is

**Review by scaustin34 — APPROVED** [review]
